packages/db/tests/test-config.js

```diff
@@ -442,7 +442,7 @@ test('canSafelyConvertBigInt() throws exception if number exceeds safe range', (
 });
 
 test('canSafelyConvertBigInt() throws exception for non-numeric string', (t) => {
-  const nonNumericString = cryptoRandomString({ length: 10 });
+  const nonNumericString = `a${cryptoRandomString({ length: 16 })}`;
```

Comment:
Can you explain this change a bit?

Reply:
Really this is a question re: if the length increase was needed.

Reply:
except that's not even true is it, a bigint i8s more than 20 digits....... I don't know what I was thinkning